src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav-links {
+  list-style: none;
```

Comment:
Avoid using tag names for styling, as recommended by the checklist. Instead, use classes to apply styles to elements. This will make your CSS more maintainable and your selectors more specific.